integration_test/EmitVerilog/sv-interfaces.mlir

```diff
@@ -0,0 +1,44 @@
+// REQUIRES: rtl-sim
+// RUN: circt-translate %s -emit-verilog -verify-diagnostics > %t1.sv
+// RUN: verilator --lint-only --top-module top %t1.sv
+// RUN: circt-rtl-sim.py %t1.sv --cycles 2 2>&1 | FileCheck %s
+
+module {
```

Comment:
Random FYI but I think you can omit the top level module declaration from test cases, which reduces indentation in the file.